e2e-tests/README.md

```diff
@@ -9,21 +15,56 @@ curl -s https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz \
     | tar -C /usr/bin --strip-components 1 -zxvpf - '*/helm'
 curl https://sdk.cloud.google.com | bash
 ```
-MacOS
+
+### MacOS
+
 ```
 brew install coreutils gnu-sed jq kubernetes-cli openshift-cli kubernetes-helm percona-xtrabackup yq
```

Comment:
`yq ` v3 should be installed now , we need to mention it because v4 does not work with our code